PocketKit/Tests/SaveToPocketKitTests/SaveToAddTagsViewModelTests.swift

```diff
@@ -104,9 +117,68 @@ class SaveToAddTagsViewModelTests: XCTestCase {
             }
         ) { _ in
         }
+        XCTAssertEqual(viewModel.recentTags, [])
+    }
+
+    func test_recentTags_withMoreThanThreeTags_andPremiumUser_returnsRecentTags() throws {
+        let item = space.buildSavedItem(tags: [])
+        try space.save()
+        let viewModel = subject(
+            item: space.viewObject(with: item.objectID) as! SavedItem,
+            user: MockUser(status: .premium),
+            retrieveAction: { _ in
+                var tags: [Tag] = []
+                for index in 1...4 {
```

Comment:
This seems to be building a different array of tags than the assertion is expecting (which is a total of 3, but we're generating 4). Why is that?

Reply:
I wanted to test that even if there are four tags, the recent tags should only gather three at most.